main.go

```diff
@@ -98,6 +67,9 @@ func ndkHome() string {
 	if v := os.Getenv(androidNDKHome); v != "" {
 		return v
 	}
+	if v := os.Getenv("ANDROID_SDK_ROOT"); v != "" {
```

Comment:
Shouldn't `ANDROID_HOME` take precedence here?
https://developer.android.com/studio/command-line/variables

Reply:
It should

Reply:
Fixed